netty/src/main/java/io/grpc/netty/NettyServerBuilder.java

```diff
@@ -227,17 +236,57 @@ public NettyServerBuilder maxHeaderListSize(int maxHeaderListSize) {
     return this;
   }
 
+  /**
+   * Sets a custom KEEPALIVE_TIME. An unreasonably small value will be bumped to a reasonable value,
```

Comment:
Don't use KEEPALIVE_TIME casing, since that is meaningless to the user. You can just say "keepalive time". But you should also define what that means. Same with timeout.